xapian-applications/omega/handler_poppler.cc

```diff
@@ -18,12 +18,9 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
  * USA
  */
-
-#include <config.h>
```

Comment:
This needs to stay - when using autotools, `<config.h>` should be included by every `.cc` file (and it should be the first header included).